index.js

```diff
@@ -1,15 +1,23 @@
-require('./lib/array_fill');
-require('./lib/array_intersect');
-require('./lib/array_qsort');
-require('./lib/array_unique');
-require('./lib/fs_scandir');
-require('./lib/object_watch');
-require('./lib/string_format');
-require('./lib/string_hashcode');
-require('./lib/string_pipe');
+module.exports = function (opts) {
+  opts = opts || {};
 
-module.exports = {
-  flatten : require('./lib/flatten'),
-  get     : require('./lib/get_deep'),
-  set     : require('./lib/set_deep')
+  require('./lib/array_fill');
+  require('./lib/array_intersect');
+  require('./lib/array_qsort');
+  require('./lib/array_unique');
+  require('./lib/fs_scandir');
+  require('./lib/object_watch');
+  require('./lib/string_format');
+  require('./lib/string_hashcode');
+
+  // skip prototype pollution of methods used by browserify
+  if (typeof opts.build == 'undefined' || opts.build == false) {
```

Comment:
When prefixed with `!`, all `undefined`, `null` & `false` evaluate to `true`. Because `opts` is supposed to always be an object, this expression can be shortened to the following without 'weird' behavior on other values:

```js
...
if (!opts.build) {
...
```